content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/ios.md

```diff
@@ -294,7 +294,7 @@ DDDatadog.verbosityLevel = DDSDKVerbosityLevelDebug;
 {{% /tab %}}
 {{< /tabs >}}
 
-3. Datadog tracer implements the [Open Tracing standard][8]. Configure and enable the shared an Open Tracing `Tracer` as `Tracer.shared()`:
+3. Datadog tracer implements both [Open Tracing][8] and [Open Telemetry][12] standards. Configure and enable the shared an Open Tracing `Tracer` as `Tracer.shared()`:
```

Comment:
Is it necessary to note OpenTracing? We've had some customer confusion about this because the project is deprecated in favor of OpenTelemetry.

Reply:
We have not deprecated the support in the SDK so it still makes sense to keep it here. Once we drop the OpenTracing APIs, it should be removed.